locale/foreman_ansible.pot

```diff
@@ -895,37 +877,37 @@ msgid "Runs an Ansible playbook which contains all the roles defined for a host"
 msgstr ""
 
 #: ../lib/foreman_ansible/remote_execution.rb:27
-msgid "Ansible: Run Insights maintenance plan"
-msgstr ""
-
-#: ../lib/foreman_ansible/remote_execution.rb:28
-msgid "Runs a given maintenance plan from Red Hat Access Insights given an ID."
-msgstr ""
-
-#: ../lib/foreman_ansible/remote_execution.rb:35
 msgid "Run playbook"
 msgstr ""
 
-#: ../lib/foreman_ansible/remote_execution.rb:36
+#: ../lib/foreman_ansible/remote_execution.rb:28
 msgid "Run an Ansible playbook against given hosts"
 msgstr ""
 
-#: ../lib/foreman_ansible/remote_execution.rb:41
+#: ../lib/foreman_ansible/remote_execution.rb:33
 msgid "Enable web console"
 msgstr ""
 
-#: ../lib/foreman_ansible/remote_execution.rb:42
+#: ../lib/foreman_ansible/remote_execution.rb:34
 msgid "Run an Ansible playbook to enable web console on given hosts"
 msgstr ""
 
-#: ../lib/foreman_ansible/remote_execution.rb:47
+#: ../lib/foreman_ansible/remote_execution.rb:39
 msgid "Upgrade Capsules on given hosts"
 msgstr ""
 
-#: ../lib/foreman_ansible/remote_execution.rb:48
+#: ../lib/foreman_ansible/remote_execution.rb:40
 msgid "Upgrade Capsules on given Capsule server hosts"
 msgstr ""
 
+#: ../lib/foreman_ansible/remote_execution.rb:45
+msgid "Update Capsules on given hosts"
+msgstr ""
+
+#: ../lib/foreman_ansible/remote_execution.rb:46
+msgid "Update Capsules on given Capsule server hosts"
```

Comment:
I think it's really poor practice to include downstream terminology into upstream, but it was already there so not blocking for this PR.

@adamruzicka I see you added it in c1ebb2c78233302573abd2c2cc79da0bbafa2063 but should this have been added to foreman_theme_satellite?

Reply:
It's not only terminology, the whole thing is downstream specific without any sensible upstream equivalent.

While it would make sense putting downstream things into the downstream plugin, including random pieces of functionality in a branding plugin doesn't feel exactly great either.